tasking/pulp/tasking/scheduler.py

```diff
@@ -260,110 +255,34 @@ def tick(self):
         old_timestamp = datetime.utcnow() - timedelta(seconds=constants.CELERYBEAT_LOCK_MAX_AGE)
 
         # Updating the current lock if lock is on this instance of celerybeat
-        result = CeleryBeatLock.objects(celerybeat_name=self.celerybeat_name).\
-            update(set__timestamp=datetime.utcnow())
-
-        # If current instance has lock and updated lock_timestamp, call super
-        if result == 1:
+        try:
+            celerybeat_lock = TaskLock.objects.get(name=celerybeat_name)
```

Comment:
this should be self.celerybeat_name which is defined in the __init__(). Changing it to that.